servers/shareable-lists-api/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "shareable-lists-api",
+  "main": "dist/main.js",
+  "prisma": {
+    "seed": "ts-node --emit=false prisma/seed.ts"
+  },
+  "scripts": {
+    "build": "rm -rf dist && dotenv -e .env.ci -- prisma generate && tsc && npm run build-schema-admin && npm run build-schema-public",
+    "watch": "tsc -w & nodemon",
+    "start": "npm run migrate:deploy && node dist/main.js",
+    "start:dev": "npm run migrate:deploy && npm run build && npm run watch",
+    "test": "dotenv -e .env.ci -- prisma generate && jest \"\\.spec\\.ts\"",
+    "test:local": "jest \"\\.spec\\.ts\"",
```

Comment:
If i run `pnpm run test` at the root on my machine will this still work?

Reply:
yes, it just does some unnecessary stuff if you've already done typegen so I added the other command